<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AmpouleVue</class>
 <widget class="QWidget" name="AmpouleVue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ampoule</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelEtat">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>-----</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bAllumer">
       <property name="text">
        <string>Allumer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bEteindre">
       <property name="text">
        <string>Eteindre</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
